packages/react-components/react-combobox/src/components/Combobox/useCombobox.tsx

```diff
@@ -299,5 +310,33 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLIn
     }
   }
 
+  const onClearIconClick = useEventCallback(
+    mergeCallbacks(state.clearIcon?.onClick, (ev: React.MouseEvent<HTMLSpanElement>) => {
+      ev.stopPropagation();
+
+      triggerRef.current?.focus();
```

Comment:
let's discuss this since https://github.com/microsoft/fluentui/pull/30010 made some changes to avoid repeated focusing

Reply:
I removed `.focus()` from `useCombobox()`, but IMO it should be kept in `useDropdown()` as there we have a focusable button

Reply:
Is that the correct behaviour? it doesn't make sense for a keyboard user to press `enter` and have the focus moved back to the input. I don't think v0 does that either

Reply:
- In Combobox: there is an icon with aria-hidden, it's not focusable. No sense to move focus
- In Dropdown: there is a focusable `button`, so you can tab to it. Once a button clicked, it will remove a value and itself, so focus should be moved back to input

Makes sence?

Reply:
ah yeah sorry I was thinking something wrong there